test/integration/basic-server.spec.js

```diff
@@ -27,56 +27,56 @@ const Static = Dispatcher.Static
 let server = null
 
 
-  let getData = function(data,timeout) {
-    return new Promise(function(resolve, reject) {
-      if (!timeout) {
+let getData = function(data, timeout) {
+  return new Promise(function(resolve, reject) {
+    if (!timeout) {
+      resolve(data);
+    } else {
+      setTimeout(function() {
         resolve(data);
-      }else{
-        setTimeout(function(){
-          resolve(data);
-        },timeout);
-      }
-    });
-  }
+      }, timeout);
+    }
+  });
+}
 
-  describe("Basic Http Server", function() {
+describe("Basic Http Server", function() {
 
-    before(function(done){
+  before(function(done) {
 
-      this.timeout(5000)
+    this.timeout(5000)
 
-      Helpers.load(path.join(__dirname,'./package.json'))
-      let env = new Env(Helpers)
-      let namespace = new Namespace(env,Helpers)
+    Helpers.load(path.join(__dirname, './package.json'))
+    let env = new Env(Helpers)
+    let namespace = new Namespace(env, Helpers)
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Missing semicolon.